modules/apps/saved-content/saved-content-taglib/src/main/java/com/liferay/saved/content/taglib/servlet/taglib/SavedContentTag.java

```diff
@@ -173,6 +140,42 @@ protected void setAttributes(HttpServletRequest httpServletRequest) {
 		}
 	}
 
+	private String _getContentTitle(Locale locale) {
+		if (Validator.isNotNull(_contentTitle)) {
+			return _contentTitle;
+		}
```

Comment:
I agree with this refactoring, however it looks strange to return always the same `_contentTitle` when the input parameter `locale` can change. I suppose this `locale` doesn't really change so I would create an attribute `private Locale _locale` and read it from `_getContentTitle`.

Does it make sense to you?

Reply:
I do not know.
The creation of a global that will be used once instead of the use of a part of a local var ..

I have to give it a think about

Reply:
I do not see any benefit of doing it on a private method.

therefore, we are remembering the user that to generate the title we depend on the locale .

Do you have a strong opinion about it?

Reply:
No, let's go for it 🚀